src/services/resource-service.ts

```diff
@@ -62,6 +83,16 @@ export const ResourceService = {
     params?: GetResourcesParams
   ): Promise<AxiosResponse<ItemsWithCount<Attachment>>> =>
     await axiosClient.get(URLs.resources.attachments.get, { params }),
+
+  getAttachmentsQuery: (params?: GetResourcesParams) => {
+    return baseService.request<ItemsWithCount<Attachment>>({
+      method: 'GET',
+      url: getFullUrl({
+        pathname: URLs.resources.attachments.get,
+        searchParameters: params
```

Comment:
Are you sure that it's working without destructuring?
```suggestion
        searchParameters: { ...params }
```